docs/running.md

```diff
@@ -0,0 +1,93 @@
+## Running the Horovod
+
+Examples below are for Open MPI. Check your MPI documentation for arguments to the `mpirun` command on your system.
+
+Typically, one GPU will be allocated per process, so if a server has 4 GPUs you would run 4 processes. In Open MPI,
+number of processes is specified with `-np` flag.
+
+Starting with the Open MPI 3, it's important to add `-bind-to none` and `-oversubscribe` arguments. `-bind-to none`
+specifies Open MPI to not bind a training process to a single CPU core (which would hurt performance). `-oversubscribe`
+allows to run multiple training processes on a single core.
+
+The `-x` option allows to specify (`-x NCCL_DEBUG=INFO`) or copy (`-x LD_LIBRARY_PATH`) an environment variable to all
+the workers.
+
+1. To run on a machine with 4 GPUs:
+
+```bash
+$ mpirun -np 4 \
+    -bind-to none -oversubscribe \
+    -x NCCL_DEBUG=INFO -x LD_LIBRARY_PATH \
+    python train.py
+```
+
+2. To run on 4 machines with 4 GPUs each:
+
+```bash
+$ mpirun -np 16 \
+    -bind-to none -oversubscribe \
+    -x NCCL_DEBUG=INFO -x LD_LIBRARY_PATH \
+    -H server1:4,server2:4,server3:4,server4:4 \
+    python train.py
+```
+
+3. If you have RoCE or InfiniBand, we found the following `pml` and `btl_openib_receive_queues` parameters to help
```

Comment:
we found the following --> we found that the 
parameters to help --> parameters improve